Userland/Libraries/LibGfx/ImageFormats/WebPWriterLossless.cpp

```diff
@@ -306,10 +309,10 @@ static ErrorOr<void> write_VP8L_image_data(Stream& stream, Bitmap const& bitmap)
     return {};
 }
 
-ErrorOr<ByteBuffer> compress_VP8L_image_data(Bitmap const& bitmap)
+ErrorOr<ByteBuffer> compress_VP8L_image_data(Bitmap const& bitmap, bool& is_fully_opaque)
```

Comment:
I'm sure that you considered that, and you came to the conclusion it was the best solution, but I have to say I don't like out parameters. I would say that it's time to move some of that to the encoder class.
I'm just commenting to ensure maintainers will see it, and take a decision.

Reply:
East const means this is fairly easy to grep for:

```sh
% rg -l ', [A-Za-z]+& ' Userland | wc -l
     589
```

So there's no shortage of out params.

(I'm not a superfan of out params either, but this function has two parameters on the RHS, which doesn't seem so terrible to me.)

Reply:
I'd also rather not have out params, but I don't feel strongly enough to block a PR on it.